global_helpers/default_test.py

```diff
@@ -6,7 +6,7 @@
 import unittest
 
 sys.path.append(os.path.dirname(__file__))
-import panther_default as p_d  # pylint: disable=C0413
+import panther_aws_helpers as p_d  # pylint: disable=C0413
```

Comment:
Also the `disable=C0413` smells. Probably it's because it has to be bellow the `sys.path.append`, but we can fix that by moving the sys.path.append in an `__init__.py` file under the `global_helpers` directory.